### DISCUSSION TITLE
Welcome to unofficial-l4m-builds Discussions!

### DISCUSSION THREAD

**Discussion by petefoth** [petefoth/unofficial-l4m-builds#1] [Announcements]
## 👋 Welcome!
I'm using Discussions as a place to provide feedback - comments, issues, suggestions. Please keep contributions civil and relevant. I will remove any that I decide are not :)

**Comment by ThomasXZ1** [comment]
Hello petefoth
I just came across to appreciate your work.
It let me use my beloved XZ1 compact.
Huge thanks.
